src/test/kotlin/no/nav/syfo/testhelper/TestDatabase.kt

```diff
@@ -38,3 +44,37 @@ class TestDatabaseNotResponding : DatabaseInterface {
     fun stop() {
     }
 }
+
+fun DatabaseInterface.createBehandlerDialogmeldingForArbeidstaker(
+    fastlege: Fastlege,
+    arbeidstakerPersonIdent: PersonIdentNumber
+): UUID {
+    this.connection.use { connection ->
+        val behandlerDialogmelding =
+            connection.createBehandlerDialogmelding(fastlege.toNewBehandler())
+        connection.createBehandlerDialogmeldingArbeidstaker(
+            arbeidstakerPersonIdent,
+            behandlerDialogmelding.first
+        )
+        connection.commit()
+
+        return behandlerDialogmelding.second
+    }
+}
```

Comment:
Er det mulig å bruke "ekte" kode i stedet for denne?

Reply:
Skal vel være mulig å bruke BehandlerService#createBehandlerDialogmelding hvis den ikke er private. 
Litt usikker på om vi skal droppe private kun for testen, men kan vel tenkes at den også kan skal brukes til å opprette andre behandlere enn fastlege på sikt så mulig det får være greit...